english/shrooms/engage-man/toothless/enlightenment/eating-marshmellows.md

```diff
@@ -0,0 +1,21 @@
+You're feeling hungry anyway, so a little marshmellow can't hurt. You decide to eat through a good deal so you have plenty of energy.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.